### PR TITLE
Remove TYPO3 8.7 specific code

### PR DESCRIPTION
**Pull request by albig**

### CODE REVIEW

File: Classes/Hooks/UserFunc.php [hunk 1]
```diff
@@ -62,81 +62,4 @@ public function displayThumbnail(&$params)
         }
         return $output;
     }
-
```
Comment (beatrycze-volk) [inline Classes/Hooks/UserFunc.php:65]: Is this part related to TYPO3 version?

Comment (albig) [reply Classes/Hooks/UserFunc.php:65]: This getDocumentType()-function is not used in TYPO3 9.5 anymore. The protected loadDocument() was only needed by getDocumentType(). That's why I removed it.

**Review by beatrycze-volk — APPROVED** [review]

**Review by ByteParty — APPROVED** [review]
